.github/CODEOWNERS

```diff
@@ -1,106 +1,586 @@
+
+# ServiceLabel: %Data Lake Storage Gen1 %Service Attention
+#/<NotInRepo>/            @sumantmehtams
+
+# ServiceLabel: %Data Lake Storage Gen2 %Service Attention
+#/<NotInRepo>/            @sumantmehtams
+
+# ServiceLabel: %Data Lake Analytics %Service Attention
+/sdk/datalake-analytics/Microsoft.Azure.Management.DataLake.Analytics/            @idear1203
+
+# ServiceLabel: %Data Lake Store %Service Attention
+/sdk/datalake-store/Microsoft.Azure.Management.DataLake.Store/            @sumantmehtams
+
+# ServiceLabel: %Data Migration %Service Attention
+/sdk/datamigration/Microsoft.Azure.Management.DataMigration/            @rgreenMSFT
+
+# ServiceLabel: %Devtestlab %Service Attention
+/sdk/devtestlabs/Microsoft.Azure.Management.DevTestLabs/            @Tanmayeekamath
+
+# ServiceLabel: %Device Provisioning Service %Service Attention
+/sdk/deviceprovisioningservices/Microsoft.Azure.Management.DeviceProvisioningServices/            @nberdy
 
 # PRLabel: %DigitalTwins
-/sdk/digitaltwins/                              @drwill-ms @timtay-microsoft @abhipsaMisra @vinagesh @azabbasi @bikamani @barustum
+# ServiceLabel: %Digital Twins %Service Attention
+/sdk/digitaltwins/                              @drwill-ms @timtay-microsoft @abhipsaMisra @vinagesh @azabbasi @bikamani @barustum @sourabhguha @inesk-vt
 
+# ServiceLabel: %Event Grid %Service Attention
 # PRLabel: %Event Grid
-/sdk/eventgrid/                                 @Kishp01 @ahamad-MS
+/sdk/eventgrid/                                 @Kishp01 @ahamad-MS @jfggdl
 
 # PRLabel: %Event Hubs
 /sdk/eventhub/                                  @serkantkaraca @jsquire
 
-# PRLabel: %Azure.Identity
-/sdk/identity/                                  @schaabs @alexanderSher
+# ServiceLabel: %Event Hubs %Service Attention
+/sdk/eventhub/Microsoft.*/           @serkantkaraca @samuelkoppes
+
+# ServiceLabel: %Functions %Service Attention
+#/<NotInRepo>/            @ahmedelnably @fabiocav
+
+# ServiceLabel: %Graph.Microsoft %Service Attention
+#/<NotInRepo>/            @dkershaw10 @baywet
+
+# ServiceLabel: %Guest Configuration %Service Attention
+/sdk/guestconfiguration/Microsoft.Azure.Management.GuestConfiguration/            @mgreenegit @vivlingaiah
+
+# ServiceLabel: %HDInsight %Service Attention
+/sdk/hdinsight/            @aim-for-better @idear1203 @deshriva
+
+# ServiceLabel: %HPC Cache %Service Attention
+#/<NotInRepo>/            @romahamu @omzevall
+
+# ServiceLabel: %Import Export %Service Attention
+#/<NotInRepo>/            @madhurinms
 
 # PRLabel: %Iot
 /sdk/iot/                                       @drwill-ms @timtay-microsoft @abhipsaMisra @vinagesh @azabbasi @prmathur-microsoft @bikamani @barustum
 
+# ServiceLabel: %IotCentral %Service Attention
+/sdk/iotcentral/Microsoft.Azure.Management.IotCentral/           @iluican @jlian
+
+# ServiceLabel: %IotDPS %Service Attention
+#/<NotInRepo>/            @iluican @jlian
+
+# ServiceLabel: %IotHub %Service Attention
```

Comment:
There's also a track two for the various  IoT bits under development at `sdk/iot/` - should we include that here?

Reply:
I think we should -- do you know which ones to add here?

Reply:
I do not, but @drwill-ms may have some insight to offer.

Reply:
I think the track1 control-plane SDKs have been owned by @iluican. We shouldn't have started track2 control-plan yet AFAIK.

We have an effort for hub data-plane track2 SDK. That would be my team of usual suspects:  @drwill-ms @timtay-microsoft @abhipsaMisra @vinagesh @azabbasi @bikamani @barustum @jamdavi